packages/eslint-plugin/utils.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * ESLint plugin: utils used in multiple rules.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+	isDependencyBlock: ( jsdoc ) => {
+		if ( jsdoc && jsdoc.description && (
+			jsdoc.description.trim() === 'Node dependencies' ||
+      jsdoc.description.trim() === 'External dependencies' ||
+      jsdoc.description.trim() === 'WordPress dependencies' ||
+      jsdoc.description.trim() === 'Internal dependencies'
+		) ) {
+			return true;
+		}
```

Comment:
Simplified a bit I think, as well as always returning a `bool`.
```suggestion
		return !! (
			jsdoc &&
			jsdoc.description &&
			/^(Node|External|WordPress|Internal) dependencies$/.test( jsdoc.description.trim() )
		);
```